src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.tsx

```diff
@@ -18,15 +18,14 @@ import React from 'react';
 import classNames from 'classnames';
 import { scaleLinear, ScaleLinear } from 'd3';
 
-import { ProteinDomain } from 'src/shared/types/thoas/product';
-
 import styles from './ProteinDomainImage.scss';
+import { FamilyMatchesProduct } from 'src/content/app/entity-viewer/state/api/queries/proteinDomainsQuery';
```

Comment:
Import order is not right

Reply:
Done (i hope i got it in the right place, there wasnt much import).